scripts/initialize/download_data.sh

```diff
@@ -11,9 +11,13 @@ synapse get syn4976369
 mv EB++AdjustPANCAN_IlluminaHiSeq_RNASeqV2.geneExp.tsv \
 data/raw/pancan_normalized_rnaseq.tsv
 
-# Clinical sample freeze info (downloaded 11 Jan 2017)
+# Clinical sample freeze info (downloaded 7 March 2017)
 # Google doc located here
 # https://docs.google.com/spreadsheets/d/1Z1H3mXdO_sk9nc0v8df7VNS_XzXiu6vKGJRbK1qYoh4/edit?usp=gmail
+# Move to `data/raw/sampleset_freeze_version4_modify.csv`
+# Modify sample freeze internally:
+# Replace '01' to '02' in the tumor code following four samples:
+# 'TCGA-06-0152-02', 'TCGA-06-0171-02', 'TCGA-06-0221-02', 'TCGA-14-0736-02'
 
 # MC3 Mutation data (Must be TCGA Jamboree User - NIHEXT)
 # Accessible throgh sftp with USER@tcgaftps.nci.nih.gov
```

Comment:
i agree that the current solution is very suboptimal...as it stands, the pipeline is not reproducible given the data access difficulties.

> I'd suggest either let the user specify his user name at the very top and use shell script to download the data or make this instruction more obvious so that it is impossible to miss.

Since this script will change (see #8) I think it is best to make the current limitations more explicit. I will make sure that anyone who stumbles on the repository and tries to run the pipeline will see exactly where the download script breaks. I also do not want to spend much time on this issue, since I know it will change in the future.